server/platform/services/sharedchannel/sync_send.go

```diff
@@ -80,15 +88,63 @@ func (scs *Service) NotifyUserProfileChanged(userID string) {
 	notified := make(map[string]struct{})
 
 	for _, user := range scusers {
-		// update every channel + remote combination they belong to.
+		// update every user + remote combination they belong to.
```

Comment:
I think we had a bug here, as the `channelID` and `remoteID` combination would send one update per channel and remote for the same user, when we should send only one update per user and remote